internal/command/locald/locald.go

```diff
@@ -75,15 +75,21 @@ func run(cfg *config.LocalDaemon, args []string) error {
 		cmd.Env = env
 
 		if err := cmd.Start(); err != nil {
-			return err
+			return fmt.Errorf("locald %s --daemon: error starting command binary=%s args=%v: %w", mgr(cfg), binary, cmd.Args, err)
 		}
 		// and check to see that it succeeded
-		return processes.WaitReady(pidFile, time.Second, cmd.Process, log)
+		if err := processes.WaitReady(pidFile, 30*time.Second, cmd.Process, log); err != nil {
```

Comment:
This seems to be the key line - nice 👌

My only thought here is that going from one second to 30 is perhaps a bit too much? In the error case (eg if AirLock blocks process execution completely), will this then hang for 30s before terminating?

You could maybe increase it to 10 seconds by default, and potentially provide a `--timeout` command-line switch to the `connect` sub-command to increase it, if necessary?

Reply:
Ideally I think one should plum the `--wait-timeout` flag from `signadot local connect` here but priorities prevent me from doing that just now.

Yes, if AirLock or something blocks it completely or beyond `--wait-timeout`, it will hang the call to `signadot local connect`. 

I'll see if I can add the `--wait-timeout` plumbing.

Reply:
I think the timeout there should be the same thing as `--wait-timeout` as passed to `signadot local connect` (and the --wait-timeout is already there for `signadot local connect`, and has a default of 10sec).

Reply:
Sounds good 👍.

Thanks again for getting this fix done quickly!

Reply:
This is done in https://github.com/signadot/cli/pull/208/commits/d80c1a0a0951c82ccdbf68af223e1a9736582e66

Reply:
> Thanks again for getting this fix done quickly!

Thanks for your help debugging it!